### ISSUE TITLE
Create design patterns for neoplasm/cancer related terms

### ISSUE THREAD

**Issue by cmungall** [obophenotype/upheno#145]
See @drseb's question here: https://github.com/obophenotype/human-phenotype-ontology/issues/473#issuecomment-142003070

Briefly:

 * MP cancer terms typically refer to disposition
 * HP cancer terms refer to an actual material object and or process in a patient (although there are 2 related syns that pertain to disposition)

We would like to use NCIt. How? Need to be careful of ontological commitments. NCIt cancers are under 'finding'. We strictly segregate disease and phenotype. If we use the same class in both contexts (we will most likely switch NCIt classes into mondo) then trouble will ensue

**Comment by lschriml** [comment]
Why not use Disease Ontology for cancer terms ?

**Comment by pnrobinson** [comment]
Suggest: We should reference the disease entities to DO and the phenotypic feature (the actual tumour) to HP and we need to decide what semantics the NCI terms have, i.e., are they closer to HP (finding) or are they closer to DO (disease).
-Peter

Dr. med. Peter N. Robinson, MSc.
Professor of Medical Genomics
Institut für Medizinische Genetik und Humangenetik
Charité - Universitätsmedizin Berlin
Augustenburger Platz 1
13353 Berlin
Germany
+4930 450566006
Mobile: 0160 93769872
peter.robinson@charite.de
http://compbio.charite.de
http://www.human-phenotype-ontology.org
Introduction to Bio-Ontologies: http://www.crcpress.com/product/isbn/9781439836651

Von: lschriml [mailto:notifications@github.com]
Gesendet: Montag, 21. September 2015 18:21
An: obophenotype/upheno
Betreff: Re: [upheno] Create design patterns for neoplasm/cancer related terms (#145)

Why not use Disease Ontology for cancer terms ?

**Comment by lschriml** [comment]
If it is helpful, DO also contains tumor terms, our terms are aligned to
NCIt terms.

Cheers,
Lynn

**Comment by mellybelly** [comment]
NCIT is the authority here, prefer to get what we need there and all the rest of us pull from there?

**Comment by pnrobinson** [comment]
Yes, we need a strategy to (1) migrate and (2) rescue old annotations and maybe (3) wrap the terms in xrefs so as to make downstream software insulated from changes etc

Dr. med. Peter N. Robinson, MSc.
Professor of Medical Genomics
Institut für Medizinische Genetik und Humangenetik
Charité - Universitätsmedizin Berlin
Augustenburger Platz 1
13353 Berlin
Germany
+4930 450566006
Mobile: 0160 93769872
peter.robinson@charite.de
http://compbio.charite.de
http://www.human-phenotype-ontology.org
Introduction to Bio-Ontologies: http://www.crcpress.com/product/isbn/9781439836651

Von: Melissa Haendel [mailto:notifications@github.com]
Gesendet: Montag, 21. September 2015 18:31
An: obophenotype/upheno
Cc: Robinson, Peter
Betreff: Re: [upheno] Create design patterns for neoplasm/cancer related terms (#145)


NCIT is the authority here, prefer to get what we need there and all the rest of us pull from there?

**Comment by cmungall** [comment]
NCIt classes are axiomatized, links back to cells, anatomy etc (which we 
would rewire to uberon/cl)

I had also assumed that NCIt more complete for cancer. What is the long 
term plan for integration?

**Comment by lschriml** [comment]
DO is working directly with NCIt, to integrate their terms which are not
yet in DO.
We are working on a set of ~300 terms identified by NCI as potentially
novel for DO.

**Comment by drseb** [comment]
Please note that every *Neoplasm* in NCIT is a subclass of *Disease or Disorder*, which is problematic for HP/MP.

**Comment by drseb** [comment]
this is slowly progressing. @cmungall @mellybelly  do we have someone actively working on this?
